version.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020 Dominik Tuchyna
```

Comment:
This file should not be needed. You can move the content of this file to `app.py`.